app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -249,4 +249,13 @@ def label_tag_mapping_delete
       javascript_flash
     end
   end
+
+  def cat_description_with_prefix(entity, cat_description)
+    prefix = MAPPABLE_ENTITIES[entity].prefix.chomp(":")
+    "#{prefix}|#{cat_description}"
+  end
+
+  def cat_description_without_prefix(cat_description)
+    cat_description.sub(/^[^\|]+\|/, "")
```

Comment:
I think it's cleaner to do `cat_description.split("|", 2).last`, but this is fine for now